gradle-plugin/it/src/it/git-annotations/expected/kubernetes.yml

```diff
@@ -6,7 +6,7 @@ items:
   kind: Deployment
   metadata:
     annotations:
-      jkube.eclipse.org/git-url: "@matches('(?:git|ssh|https?|git@[-\\w.]+):(\\/\\/)?(.*?)(\\.git)(\\/?|\\#[-\\d\\w._]+?)$')@"
+      jkube.eclipse.org/git-url: "@matches('(?:(git|https?)[-\\w.]+):(\\/\\/)?(.*?)(\\.git)(\\/?|\\#[-\\d\\w._]+?)$')@"
```

Comment:
As agreed yesterday, not yet simple enough.

Please, simplify for just the valid cases for this scenario:

- github.com:<org>/jkube.git
- https://github.com/<org>/jkube.git

I understand that something in the lines of `(https://)?github.com(:|/)[^/]+/[^\.]\.git` should work

Reply:
Sorry, I didn't get time yesterday to update this. Let  me try your suggestion.